drivers/scheduler/k8s/k8s.go

```diff
@@ -894,12 +935,80 @@ func (k *K8s) destroyCoreObject(spec interface{}, opts map[string]bool, app *spe
 		}
 
 		logrus.Infof("[%v] Destroyed Config Map: %v", app.Key, obj.Name)
+	} else if obj, ok := spec.(*ap_api.AutopilotRule); ok {
+		err := k8sOps.DeleteAutopilotRule(obj.Name)
+		if err != nil {
+			return pods, &scheduler.ErrFailedToDestroyApp{
+				App:   app,
+				Cause: fmt.Sprintf("Failed to destroy AutopilotRule: %v. Err: %v", obj.Name, err),
+			}
+		}
+
+		logrus.Infof("[%v] Destroyed AutopilotRule: %v", app.Key, obj.Name)
 	}
 
 	return pods, nil
 
 }
 
+func (k *K8s) createAutopilotObject(apParams *scheduler.AutopilotParameters) (*ap_api.AutopilotRule, error) {
+	obj := &ap_api.AutopilotRule{}
+	apRuleParams := apParams.AutopilotRuleParameters
+
+	if apRuleParams.PVCSize == 0 || apRuleParams.PVCWorkloadSize == 0 || apRuleParams.PVCPercentageUsage == 0 || apRuleParams.PVCPercentageScale == 0 {
+		return nil, fmt.Errorf("one of the autopilot rule parameter is 0")
+	}
+	obj.Name = apParams.Name
+	obj.Namespace = apParams.Namespace
+	obj.Labels = map[string]string{"creater": "torpedo"}
+	obj.Spec.PollInterval = apParams.PollInterval
+	obj.Spec.ActionsCoolDownPeriod = apRuleParams.ActionsCoolDownPeriod
+	obj.Spec.Selector.LabelSelector.MatchLabels = apRuleParams.MatchLabels //map[string]string{"name": "pgbench-data"}
+	obj.Spec.NamespaceSelector.LabelSelector.MatchLabels = map[string]string{"creater": "torpedo"}
```

Comment:
change to `creator`.

`creater` was a spelling error originally in torpedo. We should fix that and also fix the cleanup script that uses that typo.